linux/buildSrc/build.gradle

```diff
@@ -1,7 +1,80 @@
+// Plug-in order matters. The groovy plug-in must appear after java-gradle-plugin or some tests
+// won't be picked up.
+plugins {
```

Comment:
Versions of the plugins?  Could do with Ben Manes deps checker as well.

Reply:
Those are bundled plug-ins (https://docs.gradle.org/current/userguide/plugin_reference.html) and as far I understand automatically versioned together with Gradle.

Reply:
Yep, we should still probably try to version the plugins independent of the Gradle version (repeatable builds).

Reply:
I'm totally for repeatable builds. As far as I understand it, it's already as repeatable as it gets. The Gradle version is fixed via Gradle Wrapper (currently 5.4). As a result the bundled plugins are at whatever version Gradlew 5.4 shipped with. Or am I mistaken here?

If I specify a version there, we have to remember to update those versions if we change the Gradle version. If someone uses a different Gradle version, that's unsupported in any case.

Reply:
yeah fair enough